tbump/git_bumper.py

```diff
@@ -95,7 +95,8 @@ def compute_commands(self, new_version):
         tag_message = tag_name
         res.append(["tag", "--annotate", "--message", tag_message, tag_name])
         if self.remote_branch:
-            res.append(["push", self.remote_name, self.remote_branch, tag_name])
+            res.append(["push", self.remote_name, self.remote_branch])
+            res.append(["push", self.remote_name, tag_name])
```

Comment:
That should work! But maybe what you want is the `--atomic` flag of git push?

Reply:
Let's keep it as two commands just in case (man page says that atomic pushes requires stuff from the server)

Reply:
Sure, sounds fine!